apps/admin/integration-testing/cypress/support/auth.ts

```diff
@@ -1,14 +1,28 @@
 import { methodUrl } from '@votingworks/grout';
 
+// Importing all of @votingworks/auth causes Cypress tests to fail since Cypress doesn't seem to
+// interact well with PCSC Lite card reader code
+// eslint-disable-next-line vx/no-import-workspace-subfolders
```

Comment:
Even though I'm not using the PCSC Lite card reader code, just importing it in seemed to cause issues for Cypress